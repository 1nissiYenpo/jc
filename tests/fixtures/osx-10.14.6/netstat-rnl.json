[{"destination": "default", "gateway": "192.168.1.254", "route_flags": "UGSc", "route_refs": 83, "use": 24, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "127", "gateway": "127.0.0.1", "route_flags": "UCS", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "127.0.0.1", "gateway": "127.0.0.1", "route_flags": "UH", "route_refs": 18, "use": 1266231, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "169.254", "gateway": "link#10", "route_flags": "UCS", "route_refs": 1, "use": 0, "mtu": 1500, "netif": "en0", "expire": "!", "kind": "route"}, {"destination": "192.168.1", "gateway": "link#10", "route_flags": "UCS", "route_refs": 12, "use": 0, "mtu": 1500, "netif": "en0", "expire": "!", "kind": "route"}, {"destination": "192.168.1.64", "gateway": "e0:33:8e:68:38:d6", "route_flags": "UHLWIi", "route_refs": 2, "use": 470, "mtu": 1500, "netif": "en0", "expire": "753", "kind": "route"}, {"destination": "192.168.1.72", "gateway": "f0:18:98:3:d8:39", "route_flags": "UHLWIi", "route_refs": 1, "use": 8878, "mtu": 1500, "netif": "en0", "expire": "1105", "kind": "route"}, {"destination": "192.168.1.75", "gateway": "4c:56:9d:5f:b7:4c", "route_flags": "UHLWIi", "route_refs": 1, "use": 600, "mtu": 1500, "netif": "en0", "expire": "673", "kind": "route"}, {"destination": "192.168.1.80", "gateway": "0:90:a9:ed:e4:35", "route_flags": "UHLWIi", "route_refs": 1, "use": 265604, "mtu": 1500, "netif": "en0", "expire": "931", "kind": "route"}, {"destination": "192.168.1.88", "gateway": "c8:d0:83:cd:e3:2d", "route_flags": "UHLWIi", "route_refs": 1, "use": 4310, "mtu": 1500, "netif": "en0", "expire": "278", "kind": "route"}, {"destination": "192.168.1.89", "gateway": "d0:3:4b:3b:28:d5", "route_flags": "UHLWIi", "route_refs": 1, "use": 4588, "mtu": 1500, "netif": "en0", "expire": "1132", "kind": "route"}, {"destination": "192.168.1.186", "gateway": "50:32:37:d7:f5:9b", "route_flags": "UHLWIi", "route_refs": 1, "use": 6123, "mtu": 1500, "netif": "en0", "expire": "1181", "kind": "route"}, {"destination": "192.168.1.216", "gateway": "3c:37:86:15:ad:f7", "route_flags": "UHLWI", "route_refs": 0, "use": 16, "mtu": 1500, "netif": "en0", "expire": "565", "kind": "route"}, {"destination": "192.168.1.221/32", "gateway": "link#10", "route_flags": "UCS", "route_refs": 1, "use": 0, "mtu": 1500, "netif": "en0", "expire": "!", "kind": "route"}, {"destination": "192.168.1.221", "gateway": "a4:83:e7:2d:62:8f", "route_flags": "UHLWI", "route_refs": 0, "use": 36, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "192.168.1.242", "gateway": "50:14:79:12:42:3e", "route_flags": "UHLWI", "route_refs": 0, "use": 0, "mtu": 1500, "netif": "en0", "expire": "1182", "kind": "route"}, {"destination": "192.168.1.251", "gateway": "14:60:cb:10:ec:17", "route_flags": "UHLWI", "route_refs": 0, "use": 54, "mtu": 1500, "netif": "en0", "expire": "1157", "kind": "route"}, {"destination": "192.168.1.253", "gateway": "3c:37:86:15:dd:b3", "route_flags": "UHLWI", "route_refs": 0, "use": 3300, "mtu": 1500, "netif": "en0", "expire": "839", "kind": "route"}, {"destination": "192.168.1.254/32", "gateway": "link#10", "route_flags": "UCS", "route_refs": 1, "use": 0, "mtu": 1500, "netif": "en0", "expire": "!", "kind": "route"}, {"destination": "192.168.1.254", "gateway": "fc:ae:34:a1:3a:80", "route_flags": "UHLWIir", "route_refs": 29, "use": 69464, "mtu": 1500, "netif": "en0", "expire": "1200", "kind": "route"}, {"destination": "192.168.1.255", "gateway": "ff:ff:ff:ff:ff:ff", "route_flags": "UHLWbI", "route_refs": 0, "use": 4, "mtu": 1500, "netif": "en0", "expire": "!", "kind": "route"}, {"destination": "192.168.71", "gateway": "link#20", "route_flags": "UC", "route_refs": 3, "use": 0, "mtu": 1500, "netif": "vmnet8", "expire": "!", "kind": "route"}, {"destination": "192.168.71.160", "gateway": "link#20", "route_flags": "UHLWIi", "route_refs": 1, "use": 1708, "mtu": 1500, "netif": "vmnet8", "expire": "!", "kind": "route"}, {"destination": "192.168.71.255", "gateway": "ff:ff:ff:ff:ff:ff", "route_flags": "UHLWbI", "route_refs": 0, "use": 4, "mtu": 1500, "netif": "vmnet8", "expire": "!", "kind": "route"}, {"destination": "192.168.101", "gateway": "link#19", "route_flags": "UC", "route_refs": 2, "use": 0, "mtu": 1500, "netif": "vmnet1", "expire": "!", "kind": "route"}, {"destination": "192.168.101.255", "gateway": "ff:ff:ff:ff:ff:ff", "route_flags": "UHLWbI", "route_refs": 0, "use": 4, "mtu": 1500, "netif": "vmnet1", "expire": "!", "kind": "route"}, {"destination": "224.0.0/4", "gateway": "link#10", "route_flags": "UmCS", "route_refs": 2, "use": 0, "mtu": 1500, "netif": "en0", "expire": "!", "kind": "route"}, {"destination": "224.0.0.251", "gateway": "1:0:5e:0:0:fb", "route_flags": "UHmLWI", "route_refs": 0, "use": 312, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "239.255.255.250", "gateway": "1:0:5e:7f:ff:fa", "route_flags": "UHmLWI", "route_refs": 0, "use": 9918, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "255.255.255.255/32", "gateway": "link#10", "route_flags": "UCS", "route_refs": 0, "use": 0, "mtu": 1500, "netif": "en0", "expire": "!", "kind": "route"}, {"destination": "default", "gateway": "fe80::feae:34ff:fea1:3a80%en0", "route_flags": "UGc", "route_refs": 7, "use": 0, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "default", "gateway": "fe80::%utun0", "route_flags": "UGcI", "route_refs": 0, "use": 0, "mtu": 2000, "netif": "utun0", "kind": "route"}, {"destination": "default", "gateway": "fe80::%utun1", "route_flags": "UGcI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun1", "kind": "route"}, {"destination": "default", "gateway": "fe80::%utun2", "route_flags": "UGcI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun2", "kind": "route"}, {"destination": "default", "gateway": "fe80::%utun3", "route_flags": "UGcI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun3", "kind": "route"}, {"destination": "::1", "gateway": "::1", "route_flags": "UHL", "route_refs": 1, "use": 4513, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "2600:1700:bab0:d40::/64", "gateway": "link#10", "route_flags": "UC", "route_refs": 2, "use": 0, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "2600:1700:bab0:d40::1", "gateway": "fc:ae:34:a1:3a:80", "route_flags": "UHLWIi", "route_refs": 12, "use": 68412, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "2600:1700:bab0:d40::39", "gateway": "a4:83:e7:2d:62:8f", "route_flags": "UHL", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "2600:1700:bab0:d40:1874:4566:6499:f3d1", "gateway": "a4:83:e7:2d:62:8f", "route_flags": "UHL", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "2600:1700:bab0:d40:5894:f4c5:a982:26be", "gateway": "a4:83:e7:2d:62:8f", "route_flags": "UHL", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "2600:1700:bab0:d40:c9de:af8a:762c:422c", "gateway": "a4:83:e7:2d:62:8f", "route_flags": "UHL", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "2600:1700:bab0:d40:edd2:2cbf:f03a:d14f", "gateway": "a4:83:e7:2d:62:8f", "route_flags": "UHL", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "2600:1700:bab0:d40:f078:690e:f0ba:dfb", "gateway": "a4:83:e7:2d:62:8f", "route_flags": "UHL", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::%lo0/64", "gateway": "fe80::1%lo0", "route_flags": "UcI", "route_refs": 1, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::1%lo0", "gateway": "link#1", "route_flags": "UHLI", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::%en5/64", "gateway": "link#8", "route_flags": "UCI", "route_refs": 2, "use": 0, "mtu": 1500, "netif": "en5", "kind": "route"}, {"destination": "fe80::aede:48ff:fe00:1122%en5", "gateway": "ac:de:48:0:11:22", "route_flags": "UHLI", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::aede:48ff:fe33:4455%en5", "gateway": "ac:de:48:33:44:55", "route_flags": "UHLWIi", "route_refs": 77, "use": 826, "mtu": 1500, "netif": "en5", "kind": "route"}, {"destination": "fe80::%en0/64", "gateway": "link#10", "route_flags": "UCI", "route_refs": 13, "use": 0, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::df:eea7:d8e0:237a%en0", "gateway": "14:60:cb:10:ec:17", "route_flags": "UHLWI", "route_refs": 0, "use": 293, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::472:ae33:7f74:8baf%en0", "gateway": "b4:18:d1:9d:bc:2d", "route_flags": "UHLWI", "route_refs": 0, "use": 2, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::c73:1ab9:79c2:c193%en0", "gateway": "4c:56:9d:5f:b7:4c", "route_flags": "UHLWI", "route_refs": 0, "use": 1597, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::ced:5f18:1d1e:2d6b%en0", "gateway": "14:60:cb:10:ec:17", "route_flags": "UHLWI", "route_refs": 0, "use": 14, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::cf9:ca6f:7d7a:50a2%en0", "gateway": "a4:83:e7:2d:62:8f", "route_flags": "UHLI", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::10f2:d51c:68e3:bfbb%en0", "gateway": "e0:33:8e:68:38:d6", "route_flags": "UHLWI", "route_refs": 0, "use": 656, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::1406:1bd5:a957:6df2%en0", "gateway": "50:32:37:d7:f5:9b", "route_flags": "UHLWI", "route_refs": 0, "use": 2100, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::1899:d8f6:dca5:207%en0", "gateway": "50:32:37:d7:f5:9b", "route_flags": "UHLWI", "route_refs": 0, "use": 4619, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::1c4d:12ea:5e57:24ad%en0", "gateway": "f0:18:98:3:d8:39", "route_flags": "UHLWIi", "route_refs": 1, "use": 8, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::1cff:a835:c99b:22c1%en0", "gateway": "d0:3:4b:3b:28:d5", "route_flags": "UHLWI", "route_refs": 0, "use": 477, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::4ad7:5ff:fef1:86e8%en0", "gateway": "48:d7:5:f1:86:e8", "route_flags": "UHLWI", "route_refs": 0, "use": 862, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::9eb6:54ff:fe5a:5a7c%en0", "gateway": "9c:b6:54:5a:5a:7c", "route_flags": "UHLWI", "route_refs": 0, "use": 322, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::feae:34ff:fea1:3a80%en0", "gateway": "fc:ae:34:a1:3a:80", "route_flags": "UHLWIir", "route_refs": 7, "use": 27100, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "fe80::%awdl0/64", "gateway": "link#12", "route_flags": "UCI", "route_refs": 1, "use": 0, "mtu": 1484, "netif": "awdl0", "kind": "route"}, {"destination": "fe80::b41f:b7ff:fe57:a54f%awdl0", "gateway": "b6:1f:b7:57:a5:4f", "route_flags": "UHLI", "route_refs": 1, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::%utun0/64", "gateway": "fe80::30fe:52f1:103c:c66c%utun0", "route_flags": "UcI", "route_refs": 2, "use": 0, "mtu": 2000, "netif": "utun0", "kind": "route"}, {"destination": "fe80::30fe:52f1:103c:c66c%utun0", "gateway": "link#18", "route_flags": "UHLI", "route_refs": 1, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::%utun1/64", "gateway": "fe80::aaf6:1785:571a:57a9%utun1", "route_flags": "UcI", "route_refs": 2, "use": 0, "mtu": 1380, "netif": "utun1", "kind": "route"}, {"destination": "fe80::aaf6:1785:571a:57a9%utun1", "gateway": "link#21", "route_flags": "UHLI", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::%utun2/64", "gateway": "fe80::ce02:efdc:708:5411%utun2", "route_flags": "UcI", "route_refs": 2, "use": 0, "mtu": 1380, "netif": "utun2", "kind": "route"}, {"destination": "fe80::ce02:efdc:708:5411%utun2", "gateway": "link#22", "route_flags": "UHLI", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "fe80::%utun3/64", "gateway": "fe80::1188:a032:c478:4b13%utun3", "route_flags": "UcI", "route_refs": 2, "use": 0, "mtu": 1380, "netif": "utun3", "kind": "route"}, {"destination": "fe80::1188:a032:c478:4b13%utun3", "gateway": "link#23", "route_flags": "UHLI", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "ff01::%lo0/32", "gateway": "::1", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "ff01::%en5/32", "gateway": "link#8", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1500, "netif": "en5", "kind": "route"}, {"destination": "ff01::%en0/32", "gateway": "link#10", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "ff01::%awdl0/32", "gateway": "link#12", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1484, "netif": "awdl0", "kind": "route"}, {"destination": "ff01::%utun0/32", "gateway": "fe80::30fe:52f1:103c:c66c%utun0", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 2000, "netif": "utun0", "kind": "route"}, {"destination": "ff01::%utun1/32", "gateway": "fe80::aaf6:1785:571a:57a9%utun1", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun1", "kind": "route"}, {"destination": "ff01::%utun2/32", "gateway": "fe80::ce02:efdc:708:5411%utun2", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun2", "kind": "route"}, {"destination": "ff01::%utun3/32", "gateway": "fe80::1188:a032:c478:4b13%utun3", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun3", "kind": "route"}, {"destination": "ff02::%lo0/32", "gateway": "::1", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 16384, "netif": "lo0", "kind": "route"}, {"destination": "ff02::%en5/32", "gateway": "link#8", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1500, "netif": "en5", "kind": "route"}, {"destination": "ff02::%en0/32", "gateway": "link#10", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1500, "netif": "en0", "kind": "route"}, {"destination": "ff02::%awdl0/32", "gateway": "link#12", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1484, "netif": "awdl0", "kind": "route"}, {"destination": "ff02::%utun0/32", "gateway": "fe80::30fe:52f1:103c:c66c%utun0", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 2000, "netif": "utun0", "kind": "route"}, {"destination": "ff02::%utun1/32", "gateway": "fe80::aaf6:1785:571a:57a9%utun1", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun1", "kind": "route"}, {"destination": "ff02::%utun2/32", "gateway": "fe80::ce02:efdc:708:5411%utun2", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun2", "kind": "route"}, {"destination": "ff02::%utun3/32", "gateway": "fe80::1188:a032:c478:4b13%utun3", "route_flags": "UmCI", "route_refs": 0, "use": 0, "mtu": 1380, "netif": "utun3", "kind": "route"}]
