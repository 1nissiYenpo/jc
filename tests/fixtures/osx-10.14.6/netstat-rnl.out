Routing tables

Internet:
Destination        Gateway            Flags        Refs      Use    Mtu   Netif Expire
default            192.168.1.254      UGSc           83       24   1500     en0       
127                127.0.0.1          UCS             0        0  16384     lo0       
127.0.0.1          127.0.0.1          UH             18  1266231  16384     lo0       
169.254            link#10            UCS             1        0   1500     en0      !
192.168.1          link#10            UCS            12        0   1500     en0      !
192.168.1.64       e0:33:8e:68:38:d6  UHLWIi          2      470   1500     en0    753
192.168.1.72       f0:18:98:3:d8:39   UHLWIi          1     8878   1500     en0   1105
192.168.1.75       4c:56:9d:5f:b7:4c  UHLWIi          1      600   1500     en0    673
192.168.1.80       0:90:a9:ed:e4:35   UHLWIi          1   265604   1500     en0    931
192.168.1.88       c8:d0:83:cd:e3:2d  UHLWIi          1     4310   1500     en0    278
192.168.1.89       d0:3:4b:3b:28:d5   UHLWIi          1     4588   1500     en0   1132
192.168.1.186      50:32:37:d7:f5:9b  UHLWIi          1     6123   1500     en0   1181
192.168.1.216      3c:37:86:15:ad:f7  UHLWI           0       16   1500     en0    565
192.168.1.221/32   link#10            UCS             1        0   1500     en0      !
192.168.1.221      a4:83:e7:2d:62:8f  UHLWI           0       36  16384     lo0       
192.168.1.242      50:14:79:12:42:3e  UHLWI           0        0   1500     en0   1182
192.168.1.251      14:60:cb:10:ec:17  UHLWI           0       54   1500     en0   1157
192.168.1.253      3c:37:86:15:dd:b3  UHLWI           0     3300   1500     en0    839
192.168.1.254/32   link#10            UCS             1        0   1500     en0      !
192.168.1.254      fc:ae:34:a1:3a:80  UHLWIir        29    69464   1500     en0   1200
192.168.1.255      ff:ff:ff:ff:ff:ff  UHLWbI          0        4   1500     en0      !
192.168.71         link#20            UC              3        0   1500  vmnet8      !
192.168.71.160     link#20            UHLWIi          1     1708   1500  vmnet8      !
192.168.71.255     ff:ff:ff:ff:ff:ff  UHLWbI          0        4   1500  vmnet8      !
192.168.101        link#19            UC              2        0   1500  vmnet1      !
192.168.101.255    ff:ff:ff:ff:ff:ff  UHLWbI          0        4   1500  vmnet1      !
224.0.0/4          link#10            UmCS            2        0   1500     en0      !
224.0.0.251        1:0:5e:0:0:fb      UHmLWI          0      312   1500     en0       
239.255.255.250    1:0:5e:7f:ff:fa    UHmLWI          0     9918   1500     en0       
255.255.255.255/32 link#10            UCS             0        0   1500     en0      !

Internet6:
Destination                             Gateway                         Flags        Refs      Use    Mtu    Netif Expire
default                                 fe80::feae:34ff:fea1:3a80%en0   UGc             7        0   1500      en0       
default                                 fe80::%utun0                    UGcI            0        0   2000    utun0       
default                                 fe80::%utun1                    UGcI            0        0   1380    utun1       
default                                 fe80::%utun2                    UGcI            0        0   1380    utun2       
default                                 fe80::%utun3                    UGcI            0        0   1380    utun3       
::1                                     ::1                             UHL             1     4513  16384      lo0       
2600:1700:bab0:d40::/64                 link#10                         UC              2        0   1500      en0       
2600:1700:bab0:d40::1                   fc:ae:34:a1:3a:80               UHLWIi         12    68412   1500      en0       
2600:1700:bab0:d40::39                  a4:83:e7:2d:62:8f               UHL             0        0  16384      lo0       
2600:1700:bab0:d40:1874:4566:6499:f3d1  a4:83:e7:2d:62:8f               UHL             0        0  16384      lo0       
2600:1700:bab0:d40:5894:f4c5:a982:26be  a4:83:e7:2d:62:8f               UHL             0        0  16384      lo0       
2600:1700:bab0:d40:c9de:af8a:762c:422c  a4:83:e7:2d:62:8f               UHL             0        0  16384      lo0       
2600:1700:bab0:d40:edd2:2cbf:f03a:d14f  a4:83:e7:2d:62:8f               UHL             0        0  16384      lo0       
2600:1700:bab0:d40:f078:690e:f0ba:dfb   a4:83:e7:2d:62:8f               UHL             0        0  16384      lo0       
fe80::%lo0/64                           fe80::1%lo0                     UcI             1        0  16384      lo0       
fe80::1%lo0                             link#1                          UHLI            0        0  16384      lo0       
fe80::%en5/64                           link#8                          UCI             2        0   1500      en5       
fe80::aede:48ff:fe00:1122%en5           ac:de:48:0:11:22                UHLI            0        0  16384      lo0       
fe80::aede:48ff:fe33:4455%en5           ac:de:48:33:44:55               UHLWIi         77      826   1500      en5       
fe80::%en0/64                           link#10                         UCI            13        0   1500      en0       
fe80::df:eea7:d8e0:237a%en0             14:60:cb:10:ec:17               UHLWI           0      293   1500      en0       
fe80::472:ae33:7f74:8baf%en0            b4:18:d1:9d:bc:2d               UHLWI           0        2   1500      en0       
fe80::c73:1ab9:79c2:c193%en0            4c:56:9d:5f:b7:4c               UHLWI           0     1597   1500      en0       
fe80::ced:5f18:1d1e:2d6b%en0            14:60:cb:10:ec:17               UHLWI           0       14   1500      en0       
fe80::cf9:ca6f:7d7a:50a2%en0            a4:83:e7:2d:62:8f               UHLI            0        0  16384      lo0       
fe80::10f2:d51c:68e3:bfbb%en0           e0:33:8e:68:38:d6               UHLWI           0      656   1500      en0       
fe80::1406:1bd5:a957:6df2%en0           50:32:37:d7:f5:9b               UHLWI           0     2100   1500      en0       
fe80::1899:d8f6:dca5:207%en0            50:32:37:d7:f5:9b               UHLWI           0     4619   1500      en0       
fe80::1c4d:12ea:5e57:24ad%en0           f0:18:98:3:d8:39                UHLWIi          1        8   1500      en0       
fe80::1cff:a835:c99b:22c1%en0           d0:3:4b:3b:28:d5                UHLWI           0      477   1500      en0       
fe80::4ad7:5ff:fef1:86e8%en0            48:d7:5:f1:86:e8                UHLWI           0      862   1500      en0       
fe80::9eb6:54ff:fe5a:5a7c%en0           9c:b6:54:5a:5a:7c               UHLWI           0      322   1500      en0       
fe80::feae:34ff:fea1:3a80%en0           fc:ae:34:a1:3a:80               UHLWIir         7    27100   1500      en0       
fe80::%awdl0/64                         link#12                         UCI             1        0   1484    awdl0       
fe80::b41f:b7ff:fe57:a54f%awdl0         b6:1f:b7:57:a5:4f               UHLI            1        0  16384      lo0       
fe80::%utun0/64                         fe80::30fe:52f1:103c:c66c%utun0 UcI             2        0   2000    utun0       
fe80::30fe:52f1:103c:c66c%utun0         link#18                         UHLI            1        0  16384      lo0       
fe80::%utun1/64                         fe80::aaf6:1785:571a:57a9%utun1 UcI             2        0   1380    utun1       
fe80::aaf6:1785:571a:57a9%utun1         link#21                         UHLI            0        0  16384      lo0       
fe80::%utun2/64                         fe80::ce02:efdc:708:5411%utun2  UcI             2        0   1380    utun2       
fe80::ce02:efdc:708:5411%utun2          link#22                         UHLI            0        0  16384      lo0       
fe80::%utun3/64                         fe80::1188:a032:c478:4b13%utun3 UcI             2        0   1380    utun3       
fe80::1188:a032:c478:4b13%utun3         link#23                         UHLI            0        0  16384      lo0       
ff01::%lo0/32                           ::1                             UmCI            0        0  16384      lo0       
ff01::%en5/32                           link#8                          UmCI            0        0   1500      en5       
ff01::%en0/32                           link#10                         UmCI            0        0   1500      en0       
ff01::%awdl0/32                         link#12                         UmCI            0        0   1484    awdl0       
ff01::%utun0/32                         fe80::30fe:52f1:103c:c66c%utun0 UmCI            0        0   2000    utun0       
ff01::%utun1/32                         fe80::aaf6:1785:571a:57a9%utun1 UmCI            0        0   1380    utun1       
ff01::%utun2/32                         fe80::ce02:efdc:708:5411%utun2  UmCI            0        0   1380    utun2       
ff01::%utun3/32                         fe80::1188:a032:c478:4b13%utun3 UmCI            0        0   1380    utun3       
ff02::%lo0/32                           ::1                             UmCI            0        0  16384      lo0       
ff02::%en5/32                           link#8                          UmCI            0        0   1500      en5       
ff02::%en0/32                           link#10                         UmCI            0        0   1500      en0       
ff02::%awdl0/32                         link#12                         UmCI            0        0   1484    awdl0       
ff02::%utun0/32                         fe80::30fe:52f1:103c:c66c%utun0 UmCI            0        0   2000    utun0       
ff02::%utun1/32                         fe80::aaf6:1785:571a:57a9%utun1 UmCI            0        0   1380    utun1       
ff02::%utun2/32                         fe80::ce02:efdc:708:5411%utun2  UmCI            0        0   1380    utun2       
ff02::%utun3/32                         fe80::1188:a032:c478:4b13%utun3 UmCI            0        0   1380    utun3       
